client/v3/config.go

```diff
@@ -130,6 +130,30 @@ type AuthConfig struct {
 	Password string `json:"password"`
 }
 
+func (cs *ConfigSpec) Clone() *ConfigSpec {
```

Comment:
Maybe add a test that ensures that Clone doesn't get outdated if any new field is added?

Reply:
Fair point. Added a test case. Thanks